commons/dev/configmaps/flyway-readmodel-purpose-configmap.yaml

```
@@ -116,3 +116,9 @@ data:
   V1.6__Grant_Access_TokenGenerationRMChecker_Purpose.sql: |-
     GRANT USAGE ON SCHEMA "${NAMESPACE}_purpose" to "${NAMESPACE}_token_generation_readmodel_checker_user";
     GRANT SELECT ON ALL TABLES IN SCHEMA "${NAMESPACE}_purpose" TO "${NAMESPACE}_token_generation_readmodel_checker_user";
+
+  V1.7__Update_NotNull_Column_RiskAnalysisAnswer_Purpose.sql: |-
+    ALTER TABLE IF EXISTS "${NAMESPACE}_purpose".purpose_risk_analysis_answer ADD COLUMN IF NOT EXISTS "value" VARCHAR[];
+    UPDATE "${NAMESPACE}_purpose".purpose_risk_analysis_answer SET value = '{}' WHERE "value" IS NULL;
+    ALTER TABLE IF EXISTS "${NAMESPACE}_purpose".purpose_risk_analysis_answer ALTER COLUMN "value" SET NOT NULL;
```

for a couple of reasons
- if the table does not exist because of a previous problem, the migration completes successfully, while it is preferable to receive an error
- the table might not exist because not correctly written in the migration, which would complete without applying the change

```suggestion
    ALTER TABLE "${NAMESPACE}_purpose".purpose_risk_analysis_answer ADD COLUMN IF NOT EXISTS "value" VARCHAR[];
    UPDATE "${NAMESPACE}_purpose".purpose_risk_analysis_answer SET value = '{}' WHERE "value" IS NULL;
    ALTER TABLE "${NAMESPACE}_purpose".purpose_risk_analysis_answer ALTER COLUMN "value" SET NOT NULL;
```

---

`IF EXISTS` removed.